app/src/main/res/raw/data.json

```diff
@@ -604,6 +728,17 @@
         "topics": ["животные"]
         }}
     ],
+    [
+        {"EN":{
+            "word": "wool",
+            "topics": ["animals"],
```

Comment:
I think we need to remove this word because of vegans

Reply:
removed